### PR TITLE
Add analytics to tidyverse.org.

### PR DESCRIPTION
**Pull request by batpigandme**
* Add UA to `config.toml`.
* Add Google-Tag-Manager script to `head.html`.
* Closes #199.

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ relativeurls = false
 languageCode = "en-us"
 title = "Tidyverse"
 description = "An opinionated collection of R packages designed for data science."
-googleAnalytics = ""
+googleAnalytics = "UA-115082821"
```
Comment (hadley) [inline config.toml]: It might be less confusing to remove this line altogether since our custom theme no longer uses it

Comment (batpigandme) [reply config.toml]: Oh…ok. I thought GA and GTM were actually different. Will change.

Comment (hadley) [reply config.toml]: Oh maybe they are?

Comment (hadley) [reply config.toml]: Oh they are different. Do we need both?

**Pull request merged**